try {
  // code
  throw new Error('Das ist ist ein Fehler');
} catch (error) {
  // error handling
  console.log(error);
}
